pkg/generate/app/templatelookup.go

```diff
@@ -57,9 +61,13 @@ func (r TemplateSearcher) Search(terms ...string) (ComponentMatches, error) {
 						Score:       score,
 						Template:    template,
 					})
+					break
 				}
 			}
 		}
+		if exact {
+			break
```

Comment:
exact is never set unless StopOnMatch is set.

On Thu, Jan 21, 2016 at 11:55 AM, Ben Parees <notifications@github.com>
wrote:

> In pkg/generate/app/templatelookup.go
> <https://github.com/openshift/origin/pull/6670#discussion_r50428380>:
>
> >  				}
> >  			}
> >  		}
> > +		if exact {
> > +			break
>
> shouldn't this be checking StopOnMatch?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6670/files#r50428380>.
>